src/modules/messages/messages.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import { DatabaseService } from 'src/modules/database/database.service';
+import { Prisma } from '@prisma/client';
+
+@Injectable()
+export class MessagesService {
+  constructor(private readonly databaseService: DatabaseService){}
+  
+  async create(createMessageDto: Prisma.MessageCreateInput) {
+    return this.databaseService.message.create({data: createMessageDto});
+  }
+
+  findAll() {
+    return this.databaseService.message.findMany({});
+  }
+
+  findOne(uuid: string) {
+    return this.databaseService.message.findUnique({
```

Comment:
- Users should only able to access messages which are associated to their account